_inc/client/my-plan/checklist-progress-card/progress-bar/screen-reader-text/index.js

```diff
@@ -0,0 +1,8 @@
+/**
```

Comment:
Should this component be moved up a bit? I can imagine it being useful in other parts of the dashboard in the future, so having it somewhere less hidden could be useful?

Reply:
I added things deeper in the directory structure thinking that it might be easier to replace them with a shared component library when and if that appears.

That may be naive and I'm not sure whether the premise holds. Happy to move this up if you prefer.

Reply:
That makes sense. I am just afraid someone may think the same and add it somewhere else later on, without looking for it first. :)

Reply:
> a shared component library when and if that appears.

xrefing https://github.com/Automattic/wp-calypso/pull/32869#issuecomment-493029184

Reply:
This component is super simple and depends on core styling. I've moved it to shared `components`.